types/cesium/index.d.ts

```diff
@@ -121,11 +121,14 @@ declare namespace Cesium {
         readonly isConstant: boolean;
         propertyNames: string[];
 
+        constructor(value?: object, createPropertyCallback?: (value?: any) => void);
```

Comment:
Does the callback parameter need to be optional? https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks

Reply:
Did not know this - updated 6ada36d496ed9d93e0eeef52ee2bcc9f3305b7ec